usermods/usermod_v2_word_clock/usermod_v2_word_clock.h

```diff
@@ -423,11 +424,11 @@ class WordClockUsermod : public Usermod
     void addToConfig(JsonObject& root)
     {
       JsonObject top = root.createNestedObject("WordClockUsermod");
-      top["active"] = usermodActive;
-      top["displayItIs"] = displayItIs;
-      top["ledOffset"] = ledOffset;
-      top["Meander wiring?"] = meander;
-      top["Norddeutsch"] = nord;
+      top["Active"] = usermodActive;
```

Comment:
Do not change config parameters as they will reset settings when users update.
Please return to original values.

Reply:
Okay I will change it back :)